app/views/users/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<!--<h1>Basic HTML Form</h1>-->
+<!--<form action="/users" method="post" accept-charset="UTF-8">-->
+<!--  <input type="hidden" name="authenticity_token" value="<%#= form_authenticity_token %>">-->
+<!--  <label for="username">Username</label>-->
+<!--  <input id="username" type="text" name="user[username]"><br />-->
+<!--  <label for="email">Email</label>-->
+<!--  <input id="email" type="email" name="user[email]" ><br />-->
+<!--  <label for="password">Password</label>-->
+<!--  <input id="password" type="password" name="user[password]"> <br />-->
+<!--  <input type="submit" value="Signup">-->
+<!--</form>-->
+
+<%#= form_tag("/users", method: "post") do %>
+<%#= label_tag(:username, "Username:") %>
+<!--  <%#= text_field_tag(:username) %><br />-->
+<%#= label_tag(:email, "Email:") %>
+<!--  <%#= email_field_tag(:email) %><br />-->
+<%#= label_tag(:password, "Password:") %>
+<!--  <%#= password_field_tag(:password) %><br />-->
+<%#= submit_tag("Signup") %>
+<%# end %>
```

Comment:
- [ ] For the `/users` let's replace it with `users_path`
- [ ] `form_tag` should be written like this `<%= form_tag(users_path) do %>`